src/main/java/org/assertj/core/internal/Objects.java

```diff
@@ -932,6 +937,169 @@ public boolean areEqualToComparingOnlyGivenFields(Object actual, Object other,
     if (actual.hashCode() == other.hashCode()) throw failures.failure(info, shouldNotHaveSameHashCode(actual, other));
   }
 
+  /**
+   * Asserts that two <code>{@link Comparable}</code>s are equal by invoking
+   * <code>{@link Comparable#compareTo(Object)}</code>.<br>
```

Comment:
Is it? or I guess we are going to use the register comparator since `T` is not extending `Comparable`

Reply:
TBH, I'm not sure there is a lot of value having javadoc not methods and classes that are not exposed to users  (I certainly almost never read it and don't write it anymore).
If something is important to note, a regular comment will do.

Reply:
I didn't work on javadoc at all in this PR, all you see is the result of copy-paste (and most likely wrong for all the methods).

I fully agree that internal methods don't need trivial javadoc. I'll clean them up.